src/login/pam_systemd.c

```diff
@@ -957,11 +956,23 @@ _public_ PAM_EXTERN int pam_sm_close_session(
                 int argc, const char **argv) {
 
         const void *existing = NULL;
+        bool debug = false;
         const char *id;
         int r;
 
         assert(handle);
 
+        if (parse_argv(handle,
```

Comment:
I wonder if parse_argv() should be changed to return void, unless there are plans to return errors.

Reply:
hmm, yeah, hmm. i thought about this, but didn't want to do that in this patch or PR. And it feels as if sooner or later the function might have reason to fail, so maybe no worth dropping it. Also, this should probably have in pam_systemd and pam_systemd_home the same (there's a similar function there). 

I'd say: something to consider for a later PR...